pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -71,7 +71,8 @@ stdenv.mkDerivation rec {
     ln -s ${test-database} test/test-databases/database-v1.tar.xz
   '';
 
-  doCheck = !stdenv.hostPlatform.isDarwin && (lib.versionAtLeast gmime.version "3.0.3");
+  # Encryption tests seem to fail, and also due to unfound `.notmuch` database
+  doCheck = false;
```

Comment:
With https://github.com/NixOS/nixpkgs/pull/154547 merged and the tests now not being broken anymore, can we put the `doCheck` back to how it was before? I manually set it to true, and verified the tests now work again.

Reply:
> With #154547 merged and the tests now not being broken anymore, can we put the `doCheck` back to how it was before? I manually set it to true, and verified the tests now work again.

Added again the `doCheck` line.